src/Forms/Shared/Samples/Utility network/PerformValveIsolationTrace/readme.md

```diff
@@ -57,7 +57,7 @@ Tap on one or more features to use as filter barriers or create and set the conf
 
 ## About the data
 
-The [Naperville gas network feature service](https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleGas/FeatureServer), hosted on ArcGIS Online, contains a utility network used to run the isolation trace shown in this sample.
+The [Naperville gas](https://sampleserver7.arcgisonline.com/server/rest/services/UtilityNetwork/NapervilleGas/FeatureServer) network feature service, hosted on ArcGIS Online (authentication required: this is handled within the sample code), contains a utility network used to run the isolation trace shown in this sample.
```

Comment:
I don't think it is accurate to say the utility network service is hosted on ArcGIS Online. It is hosted on an instance of ArcGIS Enterprise exposed to the internet through a subdomain of arcgisonline.com. UN documentation says that publishing is only supported to ArcGIS Enterprise portals, unless that has changed recently.

Please confirm with the UN team, and update the doc if appropriate.

Reply:
This is 100 % correct. I updated the readmes to remove mention of AGOL.